x-pack/test/security_solution_cypress/package.json

```diff
@@ -27,6 +27,7 @@
     "cypress:investigations:run:serverless": "yarn cypress:serverless --spec './cypress/e2e/investigations/**/*.cy.ts'",
     "cypress:explore:run:serverless": "yarn cypress:serverless --spec './cypress/e2e/explore/**/*.cy.ts'",
     "cypress:changed-specs-only:serverless": "yarn cypress:serverless --changed-specs-only --env burn=5",
-    "cypress:burn:serverless": "yarn cypress:serverless --env burn=5"
+    "cypress:burn:serverless": "yarn cypress:serverless --env burn=5",
```

Comment:
Let't set `burn` to 2 instead of 5. It should be enough to verify the tests doesn't fail due to artefacts left.

Reply:
Fixed with https://github.com/elastic/kibana/pull/169311/commits/d25ab4e072d0f29a5004ea2cd09abbfdc67a5d00